### PR TITLE
Support record ssh private key and update README.md

### PR DESCRIPTION
**Pull request by jameszwang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,4 +78,16 @@ git push origin v0.0.1
 
 	`$ rosa-support delete vpc --vpc-id <vpc id> --region <region>`
 
+* Prepare bastion
+	* *--region* is required where created the vpc-id
+	* *--availability-zone* is required on which zone to launch the bastion instance
+    * *--vpc-id* is required which should be VPC id used to launch cluster
```
Comment (xueli181114) [inline README.md]: Is this expected to have less whitespaces from other lines?

File: README.md [hunk 2]
```diff
@@ -78,4 +78,16 @@ git push origin v0.0.1
...
+	* *--cidr-block* is NOT required, only IP address within CIDR block can access other resources through bastion proxy(default is 0.0.0.0/0)
+
+  `$ rosa-support create bastion --region us-east-2 --availability-zone us-east-2a --vpc-id <vpc id>`
```
Comment (xueli181114) [inline README.md]: Will there be any generated ssh key? Where will be private key be recorded? Can we specify it?

File: README.md [hunk 3]
```diff
@@ -78,4 +78,16 @@ git push origin v0.0.1
...
+
+* Destroy bastion, it is same with clean VPC,the bastion and related resources with VPC will be destroyed when clean VPC
```
Comment (xueli181114) [inline README.md]:
```suggestion
* Destroy bastion, the bastion and related resources with VPC will be destroyed when clean VPC
```

File: README.md (hunk 2 above)
Comment (jameszwang) [reply README.md]: No ssh key generated here. When using the bastion proxy, access is restricted by configuring which source IP addresses are allowed through CIDR block settings.

File: README.md (hunk 1 above)
Comment (jameszwang) [reply README.md]: Updated.

File: README.md (hunk 2 above)
Comment (jameszwang) [reply README.md]: Support record ssh private key to specific path now.

File: README.md (hunk 3 above)
Comment (jameszwang) [reply README.md]: Updated.

**Review by xueli181114 — APPROVED** [review]

**Pull request merged**